.github/workflows/build-check-install.yaml

```diff
@@ -0,0 +1,189 @@
+name: R CMD Check
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-install-check:
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.config.image }}:${{ matrix.config.tag }}
+    name: ${{ matrix.config.image }}, version ${{ matrix.config.tag }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {image: rocker/tidyverse, tag: '4.1.0'}
```

Comment:
There might be better CI performance with `{image: ghcr.io/insightsengineering/tidyverse_4.1.0_bioc_3.13, tag: latest}`
Please give that a shot and do a benchmark test.

Reply:
I couldn't get it to work I'm afraid https://github.com/Roche/synthetic.cdisc.data/runs/3191977819?check_suite_focus=true - any thoughts?

Reply:
I think I fixed it - visibility settings needed to be updated. Try now?

Reply:
Yeah that works - thanks